src/manifolds/SymplecticStiefel.jl

```diff
@@ -67,10 +67,10 @@ It is assumed that the point ``p`` is on ``\operatorname{Sp}(2n, 2n)``.
 """
 function canonical_projection(M::SymplecticStiefel{n,k}, p_Sp) where {n,k}
```

Comment:
```suggestion
function canonical_project(M::SymplecticStiefel{n,k}, p_Sp) where {n,k}
```
This name change is technically breaking but I doubt many people use that.

Reply:
yes it is, but we only introduced it here fir the first time and I do not thunk someone used it until now (since we introduced this in January.